docs/home/native-features.md

```diff
@@ -0,0 +1,114 @@
+?> **Note:** Stay tuned for new feature instructions as we update and expand this page.
+
+# Integrating Native Features
+
+One of the best ways to upgrade your progressive web app is to take advantage of web capabilites to integrate with the user's operating system. Modern web technology has enabled a whole host of ways to make your PWA behave more like a native application and interact seamlessly with the OS.
+
+This article will showcase how to enable various native functionality for your progressive web app.
+
+## Shortcuts 
+
+Application shorcuts allow your user to directly navigate to certain parts of your progressive web app directly from the operating system. When shorctuts are enabled, they can be accessed through your application's context menu. In Windows, the context menu can be opened by right-clicking on your app's icon.
```

Comment:
shorcuts --> shortcuts (second word of the first and second sentences)

Reply:
```suggestion
Application shortcuts allow your user to directly navigate to certain parts of your progressive web app directly from the operating system. When shortctuts are enabled, they can be accessed through your application's context menu. In Windows, the context menu can be opened by right-clicking on your app's icon.
```